autoload/tutor.vim

```diff
@@ -180,22 +179,10 @@ function! tutor#FollowLink(force)
         elseif a:force != 1 && match(l:target, '^@tutor:') > -1
             let l:tutor = matchstr(l:target, '@tutor:\zs.*')
             exe "Tutor ".l:tutor
-        else
-            exe "help ".l:target
```

Comment:
I think we should keep this or create a way so tutorials can link to help files explicitly.